front/src/modules/trainschedule/components/ManageTrainSchedule/Map.tsx

```diff
@@ -65,6 +71,20 @@ const Map = () => {
 
   const mapRef = useRef<MapRef | null>(null);
 
+  const getMapImageData = useCallback(() => {
+    if (mapRef.current) {
+      const canvas = mapRef.current;
+      const imageDataURL = canvas.getCanvas().toDataURL();
+      return imageDataURL;
+    }
+    return null;
+  }, [mapRef]);
+
+  useEffect(() => {
+    const mapBase = getMapImageData();
```

Comment:
I'm not sure, but don't you want to get the map image only if the map is loaded ? :)

Reply:
you're right, done